src/EFCore.PG/Properties/NpgsqlStrings.resx

```diff
@@ -195,4 +195,8 @@
     <value>Index '{name}' on table {tableName} cannot be scaffolded because it includes a column that cannot be scaffolded (e.g. enum).</value>
     <comment>Warning NpgsqlEventId.UnsupportedColumnIndexSkippedWarning string string</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="LogAutoPrepareDisabled" xml:space="preserve">
+    <value>Automatic preparation is set to zero and disabled for the connection to database '{database}' on server '{server}'.</value>
```

Comment:
I think the message itself doesn't go far enough in helping the user understand what's going on and what they should do... I'd say something like "A connection is being made without automatic preparation, consider enabling this feature to significantly improve performance. See http://www.npgsql.org/doc/prepare.html for more details.".